content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,76 @@
+where we have taken lessons learned as well as feedback and have applied them.
+
+## Current Work
+
+We have worked with the Envoy community to address numerous concerns which is a reason why dual-stack enablement has
+taken us a while to implement. We have [matched IP Family for outbound listener](https://github.com/envoyproxy/envoy/issues/16804)
+and [supported multiple addresses per listener](https://github.com/envoyproxy/envoy/issues/11184). Alex Xu has also
+been working fervently to get long outstanding issues resolve, with the ability for Envoy to have a
+[smarter way to pick endpoints for dual-stack](https://github.com/envoyproxy/envoy/issues/21640). Some of these improvements
+to Envoy, such as the ability to [enable socket options on multiple addresses](https://github.com/envoyproxy/envoy/pull/23496),
+have landed in the upcoming Istio 1.17 release (e.g. [extra source addresses on inbound clusters](https://github.com/istio/istio/pull/41618)).
+
+The Envoy API changes the team has made can be found under their site at [Listener addresses](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses)
+and [bind config](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig).
+Making sure we can have proper support at both the downstream and upstream connection for Envoy is important for realizing
+dual-stack support.
+
+In total the team has submitted over a dozen PRs to Envoy and are working on a least a half dozen more to make Envoy adoption of
+dual stack easier for Istio.
+
+Meanwhile, on the Istio side you can track the progress in [Issue #40394](https://github.com/istio/istio/issues/40394).
+Progress has slowed down a bit lately as we continue working with Envoy on various issues, however, we are hoping to get
+this resolved before Istio 1.17 launches in February.
+
+It is our hope that for Istio 1.17 we are able to get rudimentary support for dual-stack working. Iris Ding (Intel) and
+`Li Chun (Intel)` are already working with the community for getting redirection of network traffic for ambient. We will
+announce what changes we have made in Istio 1.17 shortly after its released.
+
+### Get involved
+
+While we do not have a publicly available version of Istio available for download with dual stack capabilities, we are
+excited to get this done and ready in Istio 1.17 due early 2023.
+
+We would love your feedback and if you are eager to work with us please stop by our slack channel, #dual-stack within
+the [Istio Slack](https://slack.istio.io/).
+
+_Thank you to the team that has worked on Istio dual-stack!_
+* Intel: [Steve Zhang](https://github.com/zhlsunshine), [Alex Xu](https://github.com/soulxu), [Iris Ding](https://github.com/irisdingbj)
+* F5: `Jacob Delgado`
+* `Yingchun Cai` (formerly of F5)
```

Comment:
And we are highlighting names here, while we have links above. Change these to links as well? And may need spelling updates?